### ISSUE TITLE
Docs: how to start the server?

### ISSUE THREAD

**Issue by GeorgeBekh** [palantir/python-language-server#429]
I'm sorry to ask such question, but i can't figure out how to start this server to use with emacs' "eglot" (in stdio mode basically)

I did `pip install 'python-language-server'` and `pip install 'python-language-server[all]'`

**Comment by GeorgeBekh** [comment]
**UPDATE:** Figured it out `python -m pyls`

**Comment by GeorgeBekh** [comment]
Solved in https://github.com/palantir/python-language-server/pull/430

**Comment by jgilmore** [comment]
Yes! Thank you for making this an issue. This really needs to be in the docs. I'm right now creating a bash script in ~/bin that reads like this
~/bin/python-language-server:
#!/bin/bash
python -m pyls

Of course, I suppose that could be done in one line:
#!/usr/bin/python -m pyls

But the one line version doesn't work. Fails with " /usr/bin/python: No module named  pyls"
but if I copy the same darn line down one, with /bin/bash above, it works. Go figure.

Anyway, I'm done here.

Do PLEASE add this to the docs somewhere!

**Comment by wufeng87** [comment]
And how to config the monaco editor to use this server? Any demo about this? Thanks a lot.

**Comment by doczoidberg** [comment]
@wufeng87 did you find a solution?

**Comment by gatesn** [comment]
There are some examples of connecting monaco to PYLS here: https://github.com/palantir/python-jsonrpc-server#examples

**Comment by tidux** [comment]
The script installed by `pip3 install python-language-server` is broken.

```
$ pyls
Traceback (most recent call last):
  File "/usr/local/bin/pyls", line 5, in <module>
    from pyls.__main__ import main
ModuleNotFoundError: No module named 'pyls.__main__'
```
